socorro/signature/README.rst

```diff
@@ -42,6 +42,8 @@ Examples:
 
     $ socorro-cmd fetch_crashids --num=10 | socorro-cmd signature
 
+  (Note that ``fetch_crashids`` defaults to ``--product=Firefox``.)
```

Comment:
reStructuredText and Sphinx have a note directive.

```suggestion
  .. Note::

     ``fetch_crashids`` defaults to Firefox. If you want a different product, use the ``--product`` argument.
     See ``socorro-cmd fetch_crashids --help`` for options.
```

Pretty sure that'll work inside the list correctly.